INSTALL.md

```diff
@@ -75,50 +89,17 @@
-
-  ```bash
-  $ cd /<root-dir>/
-  $ php -d memory_limit=-1 composer.phar update --prefer-dist
-  ```
-
-# Upgrading
-
-  Upgrade instructions are not written yet, they will before final version of eZ Platform. In the meantime follow upgrade instructions for eZ Publish 5.x as a guide if you already want to test your Platform code from 5.x on eZ Platform:
-  http://doc.ez.no/eZ-Platform/Upgrading/Upgrading-to-5.0/Upgrading-from-4.7-to-5.0
-
-
-## Configure the system
-
-1. *Optional* Dump your assets in your webroot:
-
-      This step is optional as it is automatically done for you when you install / update vendors via composer which
-      you did a few steps up. However during development you will need to execute these (especially last one) to get
-      assets to be updated in prod environment, so they are kept here for reference.
-
-       ```bash
-       php ezpublish/console assets:install --symlink web
-       php ezpublish/console assetic:dump --env=prod web
-       ```
-       The first command will symlink all the assets from your bundles in the `web/` folder, in a `bundles/` sub-folder.
-       The second command will generate the CSS and JavaScript files for the *prod* environement.
-
-       In those commands, "web" is the default folder. In the first two commands, --relative can be added for relative symlinks and further help is available with -h.
+3. *Optional* **Configure a VirtualHost**:
 
-       **Note:(1)** you should **not** run the *ezpublish/console* command as root, as it will generate cache files which the webserver and command line users will not be able to update or delete later. If sudo is installed, then you can run it with `-u www-data` for instance (means that you are root or another user who has sudo rights as `www-data`). Example:
-       ```bash
-       $ sudo -u www-data php ezpublish/console assets:install --symlink web
-       ```
-       **Note:(2)** if you are deploying ez platform on windows 7+, you need to run the command as Administrator to avoid the following error:
+    See: https://confluence.ez.no/display/EZP/Virtual+host+setup
 
-       > [Symfony\Component\Filesystem\Exception\IOException]
-       > Unable to create symlink due to error code 1314: 'A required privilege is not held by the client'. Do you have the required Administrator-rights?
-       > This is the first solution. But you can also edit the composer.json file by changing "symfony-assets-install": "symlink" to "symfony-assets-install": ""
 
-2. *Optional* Configure a VirtualHost:
+4. **Run installation command**:
 
-    See: https://confluence.ez.no/display/EZP/Virtual+host+setup
+    You may now complete the eZ Platform installation with ezplatform:install command, example of use:
 
+    ```bash
+    $ php ezpublish/console ezplatform:install --env prod demo-clean
+    ```
 
+You can now point your browser to the installation, to access Platform UI backend type in `/shell` in the URL.
```

Comment:
I'd split that in two sentences. You can now point your browser to the installation and browse the site. To access the Platform UI backend, use the `/shell` URL.